### ISSUE TITLE
Product price is not changing on the product page. Does reflect in cart. JQuery issue?

### ISSUE THREAD

**Issue by kory27** [organicinternet/magento-configurable-simple#270]
Hi,

Magento CE 1.9.2 is the version.

I have setup SCP for my configurable product to change the price based upon size. It is working and adds to the cart at the correct price, but the price doesn't change dynamically on the product page.

[Please visit this product page to view.](http://www.theframeroom.com/baltimore-md-panoramic-city-skyline-b-w-canvas-print-14.html)

I do see a lot of JQuery errors on the Inspect console of Chrome. This is admittedly an area of weakness in my dev game, so can anyone help me to resolve the conflict?

FYI - I turned off my Olark and the Mailchimp signup form in the footer and the issues were still there.

Thanks in advance!

**Comment by Tomasz-Silpion** [comment]
Main jQuery error there is "ReferenceError: jQuery is not defined".
First of all, could you disable mod_pagespeed there? It generates very strange output for javascript files.

**Comment by kory27** [comment]
Thanks for the insight. 

I don't recall putting Pagespeed in and don't see it in the Advanced area in Configuration.

Do you know where I can find that in  1.9.2?

Also, someone suggested I put this in my header misc. scripts area, but it didn't work either:

<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script><script>jQuery.noConflict();</script>

Thanks again for your help.

**Comment by Tomasz-Silpion** [comment]
Now I can see lot more but not jQuery loaded in the head.
Maybe you should add it to the local.xml (or page.xml) of your theme?
Here's the one from CDN.

`<default>
<reference name="head">
    <block type="core/text" name="google.cdn.jquery">
        <action method="setText">
            <text><![CDATA[<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script><script type="text/javascript">jQuery.noConflict();</script>]]>
            </text>
        </action>
    </block>
</reference>
</default>`

Maybe you could try to install this https://github.com/Creare/magento-jquery-include plugin instead.
Anyway, the topic is not SCP related at all. I could not even believe that it could generate such a mess.
If the store is production and you are loosing your sales right now you can mail me at tom@leftcurlybracket.com too for help.

**Comment by kory27** [comment]
Yeah. It's got to be the theme. I had to fix a bunch of stuff, i.e. mobile display, that should have worked out of the box. Not sure I'd recommend them to anyone at this point, but I like the look of the theme.

I'll try that extension too.

Be in touch.

**Comment by Seotoolsdeveloper** [comment]
I  Have solved This Earlier  and Its working. Contact Me via gambanet2015@gmail.com 
for more Information
